src/services/TestDataService.ts

```diff
@@ -702,16 +717,18 @@ export class TestDataService {
     async createBasicShippingMethod(
         overrides: Partial<ShippingMethod> = {},
     ): Promise<ShippingMethod> {
+        const deliveryTime = await this.getAllDeliveryTimeResources()
 
-        const deliveryTime = await this.getAllDeliveryTimeResources();
-
+        overrides.availabilityRuleId ??= (await this.getRule('Always valid (Default)')).id
```

Comment:
Is that 6.5.x relevant? I see in the docs its is [required in 6.5](https://shopware.stoplight.io/docs/admin-api/branches/v6.5/32dcb684349f4-create-a-new-shipping-method-resources). and optional in 6.6.

Reply:
yes, because it's required in 6.5, we should provide the default